examples/translator/module/rust/src/lib.rs

```diff
@@ -25,7 +25,9 @@ use translator_common::proto::{
 
 oak::entrypoint!(oak_main => {
     oak::logger::init_default();
-    TranslatorDispatcher::new(Node)
+    let dispatcher = TranslatorDispatcher::new(Node);
+    let grpc_channel = oak::grpc::server::init_default();
+    oak::run_event_loop(dispatcher, grpc_channel);
```

Comment:
This will need to be split up somehow, so it's still possible to use the `translator` Node as a library Node that's part of another Application &ndash; in the latter context, the Node instance will be handed an initial handle and *shouldn't* create a new gRPC server Node.

(If #901 weren't open, the `hello_world` example would illustrate this use case.)

Reply:
Fixed